test/indexer/test_hca_indexer.py

```diff
@@ -1251,10 +1251,29 @@ def test_accessions_fields(self):
         for hit in hits:
             contents = hit['_source']['contents']
             project = one(contents['projects'])
-            self.assertEqual(['SRP000000'], project['insdc_project_accessions'])
-            self.assertEqual(['GSE00000'], project['geo_series_accessions'])
-            self.assertEqual(['E-AAAA-00'], project['array_express_accessions'])
-            self.assertEqual(['PRJNA000000'], project['insdc_study_accessions'])
+            accessions = {
+                'array_express': ['E-AAAA-00'],
+                'geo_series': ['GSE00000'],
+                'insdc_project': ['SRP000000', 'SRP000001'],
+                'insdc_study': ['PRJNA000000']
+            }
+            self.assertEqual(accessions['insdc_project'],
+                             project['insdc_project_accessions'])
+            self.assertEqual(accessions['geo_series'],
+                             project['geo_series_accessions'])
+            self.assertEqual(accessions['array_express'],
+                             project['array_express_accessions'])
+            self.assertEqual(accessions['insdc_study'],
+                             project['insdc_study_accessions'])
+            entity_type, aggregate = self._parse_index_name(hit)
+            if entity_type == 'project':
+                expected_accesions = [
+                    {'namespace': n, 'accession': a}
+                    for n, as_ in accessions.items()
```

Comment:
Since you wrap this anyways, you can avoid `as_` by using long names for the loop vars.